resources/data/layers/default.json

```diff
@@ -274,4 +295,4 @@
       ]
     }
   ]
-}
+}
```

Comment:
Very minor: Please add newline

Reply:
Alright. Fixed in this commit 35ac69f2919081e3d922723b870c225e9c5a6afe

Reply:
Yes, thank you. Feel free to merge.